examples/read_all_values.rs

```diff
@@ -10,7 +10,7 @@ fn main() {
 
 #[cfg(feature = "experimental-reader")]
 mod example {
-    use ion_rs::result::IonResult;
+    use ion_rs::IonResult;
```

Comment:
🗺️ I have a `TODO` to limit the visibility of types to a single prescribed path within the crate. For the moment, `IonResult` can be found in both the crate root (`crate::IonResult`) and in `result` (`crate::result::IonResult`). This is true of other types as well.